content/docs/user-guide/gems/repositories/overview.md

```diff
@@ -0,0 +1,194 @@
+---
+linkTitle: Overview
+title: Gem Repository Overview
+description: Gems repositories provide a way to share Gems in Open 3D Engine (O3DE).
+weight: 100
+toc: true
+---
+
+
+Gem repositories provide a way to discover, acquire, and share Gems that are not included with **Open 3D Engine (O3DE)**.  Gem repositories are generally hosted online, but you can also add them to your project from a local hard drive which is helpful for testing.
+
+
+## Add a Gem repository in Project Manager
+
+You can add Gem repositories from the **Gem Repositories** page in Project Manager which you can access from the **Engine** tab, or from the **Gem Catalog** right panel menu.
+1. Press the **Add Repository** button. In the dialogue box that opens you can supply the URL or local path for a Gem repository. 
+1. If you have a URL for an online Gem repository, copy and paste it into the `Repository Path` field and press the **Add** button.
+1. Alternatively, if you want to add a Gem repository from your local hard drive, press the folder button to open the `Browse` dialog box, select the folder with the Gem repository, press the **Select Folder** button, and finally press the **Add** button.
+
+A new entry will be added to the list of Gem repositories, and the Gems provided by the Gem repository will be available to download and add to your projects.
+
+## Add a Gem repository with the o3de command line tool
+
+You can use the `o3de` CLI tool to add Gem repositories with the following command:
+```
+scripts\o3de.bat register -ru <Gem repository URL>
+```
+
+## Anatomy of a Gem repository  
+
+A Gem repository contains a `repo.json` file which contains the metadata for the repository, including a list of URIs for the Gems it provides.
+
+### Folder structure
+
+The recommended folder structure is to put the `repo.json` file at the root of your Gem repository and the `gem.json` files for each Gem it provides in subfolders.  
+```
+/
+   repo.json
+   ExampleGem1/
+       gem.json
+   ExampleGem2/
+       gem.json
+```
+
+If you are confident you will only ever have one Gem in the repository, it's acceptable to put the `gem.json` in the root folder.
+
+{{< tip >}}
+One of the most convenient ways to host a Gem Repository online is to place the `repo.json` file inside the GitHub repository where your Gem's source code is.
+{{< /tip >}}
+
+### Format of a repo.json file
+The `repo.json` file contains the metadata for the Gem repository and a list of Gems.
+
+```json
+{
+    "repo_name":"RepositoryName",
+    "origin":"RemoteOriginName",
+    "repo_uri": "https://o3de.org/repo/v1",
```

Comment:
Nit: I realize you just want to use a fictional url for the repo, but I would think in the examples in these next two sections you'd want it to start with `https://github.com`, to help clarify the association with repos for the reader. So for example, `https://github.com/MyRepo/` for the repo_uri and `https://github.com/MyRepo/ExampleGem/` for the link to the Gem.

Reply:
Gem repos do not have to be in GitHub, and I wasn't sure at the time of writing if that was fully supported - though it looks like it might be?  If we were to use a GitHub URL example then we'd use `https://github.com/MyUsernameOrOrg/MyRepo` for the `repo_uri` and the Gem URL would probably be something like 
`https://github.com/MyUsernameOrOrg/MyRepo/releases/download/1.0.0/ExampleGem.zip`  (with SHA256 verification) or
`https://github.com/MyUsernameOrOrg/MyRepo/archive/refs/heads/main.zip` (without SHA256 verification)